sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -230,8 +230,8 @@ async function main() {
   console.log(`Created on: ${response.createdOn}`);
   console.log(`Last modified: ${response.lastModified}`);
 
-  if (response.models) {
-    for (const submodel of response.models) {
+  if (response.submodels) {
```

Comment:
thanks @jeremymeng, can you add this change to the changelog? everything else looks good to me

Reply:
I will work on a separate PR for changelog since I didn't update it for my previous consistency renaming yet.